Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use multi-stage build
+FROM golang:1.15 as builder
+
+ARG KAVANODE_VERSION
+ARG ROSETTA_KAVA_VERSION
```

Comment:
Versioning needs to be explicit (https://www.rosetta-api.org/docs/node_deployment.html#build-anywhere).  We can hard code the versions in the dockerfile

Reply:
Updated in https://github.com/Kava-Labs/rosetta-kava/pull/10/commits/110777c79c548f618461a08d02e9243d77937d7b. For production we'll need to update the `ROSETTA_KAVA_VERSION` to an official tagged release + update the `NETWORK` used in start-services.sh to the latest mainnet ID.